m/metrics-core/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "maintainer": "mayurbhosure",
+  "package_name": "metrics",
```

Comment:
package name should match folder name. Name the package as metrics-core.
Update the script name as well.